official/utils/arg_parsers/parsers.py

```diff
@@ -201,6 +229,31 @@ def __init__(self, add_help=False, num_parallel_calls=True, inter_op=True,
           metavar="<MTS>"
       )
 
+    if dtype:
+      self.add_argument(
+          "--dtype", "-dt",
+          default="fp32",
+          choices=["fp16", "float16", "fp32", "float32", "int8"],
```

Comment:
int8 is not actually a choice currently, and will fail with a ValueError above, correct? In any case, a call to .keys() on a constant will keep us in sync.